sonar-plugin-api/src/test/java/org/sonar/api/server/ws/RequestTest.java

```diff
@@ -214,6 +222,21 @@ public void param_as_date() {
     assertThat(underTest.setParam("a_date", "2014-05-27").paramAsDate("a_date")).isEqualTo(DateUtils.parseDate("2014-05-27"));
   }
 
+  @DataProvider
+  public static Object[][] date_times() {
```

Comment:
if it was as concise as this, it would be a no brainer (apart from the Runner)
![pragmatists_junitparams__parameterised_tests_that_don_t_suck](https://cloud.githubusercontent.com/assets/1247713/18506373/1b8b2ec0-7a6c-11e6-8d73-b07497d8b5a2.png)